cmd/machine-os-builder/start.go

```diff
@@ -98,6 +100,9 @@ func runStartCmd(_ *cobra.Command, _ []string) {
 	ctrl, err := getBuildController(ctx, cb)
 	if err != nil {
 		klog.Fatalln(err)
+		if strings.Contains(err.Error(), "invalid image builder type") {
```

Comment:
suggestion: One typically should not match errors based upon the contents of its string. Instead, the two better options are to either create a sentinel error to match against or a custom error type. The reason for this is because it is possible for two errors to have the same string value but to mean completely different things and should be handled in two completely different ways. Have a look at https://earthly.dev/blog/golang-errors/ for an example of both patterns.